TYPO3.Media/Classes/TYPO3/Media/Controller/AssetController.php

```diff
@@ -397,10 +397,6 @@ public function createTagAction($label)
         $existingTag = $this->tagRepository->findByLabel($label);
         if (count($existingTag) > 0) {
             $this->addFlashMessage(sprintf('Tag "%s" already exists.', $label), '', \TYPO3\Flow\Error\Message::SEVERITY_ERROR);
-            if (($assetCollection = $this->browserState->get('activeAssetCollection')) !== null && $assetCollection->addTag($existingTag)) {
-                $this->assetCollectionRepository->update($assetCollection);
-                $this->addFlashMessage(sprintf('Tag "%s" added to collection.', $label));
-            }
```

Comment:
i see, did not see that. How will this that play out in the overview for all asset collections? Will there be duplicate tags? Could get confusing fast